### PR TITLE
Update prettier config

### PR DESCRIPTION
**Pull request by dimak1**
*Description of changes:*

Update prettier config to format the code.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]

File: ppr-ui/prettier.config.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
 module.exports = {
   endOfLine: 'lf',
   jsxBracketSameLine: false,
-  printWidth: 80,
+  printWidth: 120,
```
Comment (cameron-eyds) [inline ppr-ui/prettier.config.js:4]: This is probably a good change, is this why it broke 3 lines of code into 5?
Does this fix that?

File: ppr-ui/prettier.config.js [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
   semi: false,
   singleQuote: true,
   jsxSingleQuote: false,
   tabWidth: 2,
-  trailingComma: 'es5',
+  trailingComma: 'none',
   useTabs: false,
-  vueIndentScriptAndStyle: false
+  vueIndentScriptAndStyle: false,
+  bracketSpacing: true,
```
Comment (cameron-eyds) [inline ppr-ui/prettier.config.js:14]: Another good change i think

File: ppr-ui/prettier.config.js (hunk 1 above)
Comment (dimak1) [reply ppr-ui/prettier.config.js:4]: Yea, this helps with that. Now, if you use prettier with this config, it won't (shouldn't) break the lines unless over 120 chars.

File: ppr-ui/prettier.config.js [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
+  arrowParens: 'avoid'
```
Comment (cameron-eyds) [inline ppr-ui/prettier.config.js:15]: Also a good change 👍

**Review by cameron-eyds — APPROVED** [review]
Looks good 👍

**Pull request merged**